graylog2-web-interface/src/views/stores/isEqualForSearch.ts

```diff
@@ -17,12 +17,12 @@
 import isEqualWith from 'lodash/isEqualWith';
 import isFunction from 'lodash/isFunction';
 
-const hasFn = (obj, fn) => obj && obj[fn] && isFunction(obj[fn]);
-const hasEquals = (obj) => hasFn(obj, 'equals');
-const hasEqualsForSearch = (obj) => hasFn(obj, 'equalsForSearch');
-const isImmutable = (obj) => hasFn(obj, 'toJS');
+const hasFn = (obj: any, fn: string) => obj && obj[fn] && isFunction(obj[fn]);
```

Comment:
Would it be better to use `unknown` here?

Reply:
This is one of the rare cases where `any` really makes sense, because it is supposed to be applicable to anything. Btw, `isFunction` takes an `any` too, for the same reason IMO.